spec/jobs/autodrop_spec.rb

```diff
@@ -0,0 +1,52 @@
+require 'spec_helper'
+require 'routemaster/jobs/autodrop'
+require 'routemaster/models/database'
+require 'routemaster/models/subscriber'
+require 'routemaster/models/batch'
+require 'spec/support/persistence'
+
+describe Routemaster::Jobs::Autodrop do
+  subject { described_class.new(batch_size: 2, database: database) }
+  let(:database) { double 'database' }
```

Comment:
Any reason not to use `instance_double` here so that the tests ensure we're only using methods that actually exist?